### ISSUE TITLE
Use path variables in response

### ISSUE THREAD

**Issue by ChrisMcKenzie** [datascienceinc/accord#8]
It is possible to create an endpoint with a "path variable" in it like so `/user/{id}` however there is currently no way to use that anywhere it would be nice to use it in for example the mocked response.

I think this can be achieved by implementing [hashicorp/hil](https://github.com/hashicorp/hil) so the usage might look something like this

```
endpoint "/users/{id}" {
  method = "GET"

  response {
    headers {
      X-MY-HEADER = "stuff"
			Content-Type = "application/json"
    }

    code = 200

    body = {
      message = "hello, ${var.id}"
    }
  }
}
```

**Comment by AlecAivazis** [comment]
What are your thoughts on also support query parameters too?

**Comment by ChrisMcKenzie** [comment]
yeah, I think those would slip in to this feature as well. so you might have something like `${var.query.my-super-cool-param}`
